template/carousel/carousel.html

```diff
@@ -1,6 +1,6 @@
 <div ng-mouseenter="pause()" ng-mouseleave="play()" class="carousel" ng-swipe-right="prev()" ng-swipe-left="next()">
     <ol class="carousel-indicators" ng-show="slides.length > 1">
-        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="select(slide)"></li>
+        <li ng-repeat="slide in slides | orderBy:'index' track by $index" ng-class="{active: isActive(slide)}" ng-click="!$currentTransition && select(slide)"></li>
```

Comment:
I would go for that indeed. Just keep the `currentTransition` variable as an internal flag like suggested also keeps the template cleaner. 

We promote overwriting templates to implement custom designs/functionality. By keeping `$currentTransition` out of the template, there is one less variable to keep in mind that could potentially break a custom template.